nimble-newts/askgrieves/askgrieves/settings.py

```diff
@@ -0,0 +1,129 @@
+"""
+Django settings for askgrieves project.
+
+Generated by 'django-admin startproject' using Django 3.0.8.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'jc&@p_n-q^e4z&-l0y@jo$n+bg0y4v^vvvx=yoqjkyf-ugc+_2'
```

Comment:
These should be set to something like an environment variable set in production.

This is the secret that Django admin will use and keeping the secret key open could allow for tampering.